### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by anas599**
Describe the core principles of building accessible websites and explain why accessibility is essential.

- Page titles already no issues found
- Image text alternative no issues found
- Text headings no issues found
- Color Contrast no issues found
- Resize no issues found
- Moving content no issues found
- Multimedia no issues found
- Basic structure of the page: Added <i> tag to add the social icons on lines 31-34 and lines 172-175

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,10 +169,10 @@ <h1>Skills</h1>
...
         <footer>
             <ul class="social_form">
-                <li><a href="#" class="fa fa-facebook-square"></a></li>
-                <li><a href="#" class="fa fa-twitter-square"></a></li>
-                <li><a href="#" class="fa fa-github-square"></a></li>
-                <li><a href="#" class="fa fa-instagram"></a></li>
+            <li><a href="https://www.facebook.com/anas599/" class="fa fa-facebook-square"></a></li>
+            <li><a href="https://twitter.com/anas599" class="fa fa-twitter-square"></a></li>
+            <li><a href="https://github.com/anas599" class="fa fa-github-square"></a></li>
+            <li><a href="https://www.instagram.com/anas.sufyan93/" class="fa fa-instagram"></a></li>
```
Comment (mckent05) [inline index.html]: - Good job including links for your social media icons, but for improved accessibility, I suggest you use the `i` tag to place the icons on your page. You can refer to [this article](https://www.w3schools.com/icons/) for your info 👍 👍 
- Please do not forget to include these changes in your report, under the point `the basic structure of the page`.

File: index.html [hunk 2]
```diff
@@ -28,10 +28,10 @@ <h2>I’m a software developer</h2>
...
             If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         <ul class="social">
-            <li><a href="#" class="fa fa-facebook-square"></a></li>
-            <li><a href="#" class="fa fa-twitter-square"></a></li>
-            <li><a href="#" class="fa fa-github-square"></a></li>
-            <li><a href="#" class="fa fa-instagram"></a></li>
+            <li><a href="https://www.facebook.com/anas599/" class="fa fa-facebook-square"></a></li>
+            <li><a href="https://twitter.com/anas599" class="fa fa-twitter-square"></a></li>
+            <li><a href="https://github.com/anas599" class="fa fa-github-square"></a></li>
+            <li><a href="https://www.instagram.com/anas.sufyan93/" class="fa fa-instagram"></a></li>
```
Comment (mckent05) [inline index.html]: - Good job including links for your social media icons, but for improved accessibility, I suggest you use the `i` tag to place the icons on your page. You can refer to [this article](https://www.w3schools.com/icons/) for your info 👍 👍
- Please do not forget to include these changes in your report, under the point of `the basic structure of the page`.

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @anas599,
 
# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job including a report in the PR of your project 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @anas599   , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] [OPTIONAL] kindly  Add a meaningful description in your pull request in the future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Comment by anas599** [comment]
Changed tags to make the website more accessible friendly

**Pull request merged**